src/App.js

```diff
@@ -5,182 +5,168 @@ import Login from './Login'
 import About from './About'
 
 import { Route, Switch } from 'react-router-dom';
-import { gql, useQuery, useMutation, useLazyQuery } from '@apollo/client';
+import { gql, useMutation, useLazyQuery } from '@apollo/client';
 import DogProfile from './DogProfile'
 
 const App = () => {
 
-const [user, setUser] = useState({})
+  const [user, setUser] = useState({})
 
-const [username, setUsername] = useState('')
-const [email, setEmail] = useState('')
+  //verify we use lines 16+17 at production stage
+  const [username, setUsername] = useState('') 
```

Comment:
I like this comment, so that we can remember to come back to it. We don't want to have "orphaned" code.